File: src/main/java/pl/yalgrin/playnite/simplesync/mapper/objects/CompletionStatusMapper.java
package pl.yalgrin.playnite.simplesync.mapper.objects;

import org.springframework.stereotype.Component;
import pl.yalgrin.playnite.simplesync.library.domain.CompletionStatus;
import pl.yalgrin.playnite.simplesync.library.dto.CompletionStatusDTO;

@Component
public class CompletionStatusMapper extends AbstractObjectMapper<CompletionStatus, CompletionStatusDTO> {

    @Override
    protected CompletionStatusDTO createDTO() {
        return new CompletionStatusDTO();
    }
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/mapper/CompletionStatusMapper.kt
package pl.yalgrin.playnite.simplesync.library.mapper

import org.springframework.stereotype.Component
import pl.yalgrin.playnite.simplesync.library.domain.CompletionStatus
import pl.yalgrin.playnite.simplesync.library.dto.CompletionStatusDTO

@Component
class CompletionStatusMapper : LibraryObjectMapperImpl<CompletionStatus, CompletionStatusDTO>() {
    override fun createDTO(): CompletionStatusDTO = CompletionStatusDTO()
}